src/platform/Zephyr/BLEManagerImpl.h

```diff
@@ -138,6 +138,9 @@ class BLEManagerImpl final : public BLEManager, private BleLayer, private BlePla
 
     static BLEManagerImpl sInstance;
 
+    // The summarized number of Bluetooth LE connections related to the device (including these not related to Matter service).
+    static uint16_t sBleConnNum;
```

Comment:
Is this only used in logging of the current number of connections? BTW. The `static` is probably not needed here as the whole class is implemented as a singleton (we could spare additional definition in cpp file).

Reply:
Yes, initially I was going to remove this log, but after all it seems to be useful. Unfortunately I couldn't find the way to get from BT stack the current number of connected devices, so I had to create a counter.